implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -566,10 +566,10 @@ impl NodeManagerWorker {
                 encode_response(self.get_tcp_connection(req, address.to_string()).await)?
             }
             (Post, ["node", "tcp", "connection"]) => {
-                encode_response(self.create_tcp_connection(req, dec, ctx).await)?
+                encode_response(self.create_tcp_connection(req, dec.decode()?, ctx).await)?
```

Comment:
I forgot, could you please put the arguments in the same order as in most other places: `ctx, req, dec.decode()?`. Thanks!

Reply:
done